content/docs/release-notes/archive/2111-2-release-notes.md

```diff
@@ -0,0 +1,43 @@
+---
+linktitle: 2111.2 Release Notes
+title: Release Notes for Open 3D Engine 2111.2
+description: Find out what was delivered in O3DE 2111.2
+weight: 100
+toc: true
+---
+
+Open 3D Engine release 2111.2 is a maintenance and quality of life improvement release based on 2111.1. This release is bugfix-only and contains no new features.
+
+## Bug fixes
+
+* Duplicate engine detection and help in Project Manager [#6984](https://github.com/o3de/o3de/pull/6984)
+* Add MainPipeline to pass hierarchy for when HDR color grading component requests for the lut generation's image attachment. [#6436](https://github.com/o3de/o3de/pull/6436)
+* fixes for stability [#6434](https://github.com/o3de/o3de/pull/6434) 
+* Fixed SRG updated related crash
```

Comment:
This item and the next should be indented to show it is a subset of the PR above. Any items without a PR are subsets of the PR above them.

Reply:
Fixed; formatting error.